### PR TITLE
47 name and id

### PR DESCRIPTION
**Pull request by samvanderzwan**
Refactored the assets to use both the name and id of the esdl. Refactor these to be str instead of uuid,

### CODE REVIEW

File: src/simulator_core/entities/utility/influxdb_reader.py
Comment (octavianor) [inline src/simulator_core/entities/utility/influxdb_reader.py:1]: this is not related to change of name or id

File: src/simulator_core/solver/network/assets/base_node_item.py [hunk 1]
```diff
@@ -24,13 +23,15 @@
...
+    def __init__(self, name: str, identifier: str, number_of_unknowns: int):
         """Initializes the BaseNodeItem object with the given parameters.
 
         :param int number_of_unknowns: The number of unknown variables for the item.
-        :param uuid.UUID name: The unique identifier of the item.
+        :param str name: The name of the node.
+        :param str identifier: The unique identifier of the node.
         """
         self.name = name
+        self.id = identifier
```
Comment (octavianor) [inline src/simulator_core/solver/network/assets/base_node_item.py:34]: this is already id, the rest why identifier?

**Review by octavianor — COMMENTED** [review]
maybe just call it id, instead of identifier

File: src/simulator_core/entities/utility/influxdb_reader.py
Comment (samvanderzwan) [reply src/simulator_core/entities/utility/influxdb_reader.py:1]: Reverted this change

File: src/simulator_core/solver/network/assets/base_node_item.py (hunk 1 above)
Comment (samvanderzwan) [reply src/simulator_core/solver/network/assets/base_node_item.py]: changed everything to id or _id

**Review by octavianor — APPROVED** [review]
approve the change identifier to _id
